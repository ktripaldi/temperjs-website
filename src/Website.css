.Website {
  text-align: center;
}

.Website-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .Website-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.Website-header {
  background-color: #ffffff;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: black;
}

.Website-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
