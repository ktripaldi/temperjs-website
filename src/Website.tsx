import React from 'react'
import logo from './logo.svg'
import './Website.css'

function Website() {
  return (
    <div className="Website">
      <header className="Website-header">
        <img src={logo} className="Website-logo" alt="logo" />
        <h1>Work in progress</h1>
      </header>
    </div>
  )
}

export default Website
