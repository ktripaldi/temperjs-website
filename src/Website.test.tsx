import React from 'react'
import { render, RenderResult } from '@testing-library/react'
import Website from './Website'

describe('Forecast Result index page', () => {
  let component: RenderResult
  beforeEach(() => {
    component = render(<Website />)
  })

  it('should match the snapshot', () => {
    expect(component.asFragment()).toMatchSnapshot()
  })
})
