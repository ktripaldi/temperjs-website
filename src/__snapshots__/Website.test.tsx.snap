// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forecast Result index page should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="Website"
  >
    <header
      class="Website-header"
    >
      <img
        alt="logo"
        class="Website-logo"
        src="logo.svg"
      />
      <h1>
        Work in progress
      </h1>
    </header>
  </div>
</DocumentFragment>
`;
